tox.ini

```diff
@@ -77,4 +77,4 @@ setenv =
   READTHEDOCS_PROJECT = poliastro
   READTHEDOCS_VERSION = latest
 commands =
-    sphinx-build -d "{toxworkdir}/docs_doctree" docs/source "{toxworkdir}/docs_out" --color -vW -bhtml
+    sphinx-build -d "{toxworkdir}/docs_doctree" docs/source "{toxworkdir}/docs_out" --color -v -bhtml
```

Comment:
There are some outstanding warnings still:

```
/home/vsts/work/1/s/docs/source/user_guide.md:25: WARNING: Non-consecutive header level increase; 1 to 3
/home/vsts/work/1/s/docs/source/user_guide.md:107: WARNING: Non-consecutive header level increase; 1 to 3
/home/vsts/work/1/s/docs/source/user_guide.md:163: WARNING: Literal block expected; none found.
/home/vsts/work/1/s/docs/source/user_guide.md:181: WARNING: Literal block expected; none found.
/home/vsts/work/1/s/docs/source/user_guide.md:312: WARNING: Literal block expected; none found.
/home/vsts/work/1/s/docs/source/user_guide.md:342: WARNING: Literal block expected; none found.
/home/vsts/work/1/s/docs/source/user_guide.md:442: WARNING: Literal block expected; none found.
```

And 1 more configuration warning. Please address those.

Reply:
Sure :)